components/sup/src/manager/service/hook_runner.rs

```diff
@@ -40,7 +57,26 @@ impl<H> HookRunner<H> where H: Hook + Sync
                      pkg,
                      passwd }
     }
+
+    pub fn retriable_future(&self) -> (FutureHandle, impl Future<Item = (), Error = ()>) {
```

Comment:
"retriable" is a totally valid spelling, though it might be worth considering using "retryable", so it turns up in a search for "retry"